Formula/redis.rb

```diff
@@ -22,7 +22,9 @@ class Redis < Formula
   depends_on "openssl@1.1"
 
   def install
-    system "make", "install", "PREFIX=#{prefix}", "CC=#{ENV.cc}", "BUILD_TLS=yes"
+    system "make", "install", "PREFIX=#{prefix}", "CC=#{ENV.cc}", "BUILD_TLS=yes",
+      "OPENSSL_CFLAGS=-I#{Formula["openssl@1.1"].include}",
+      "OPENSSL_LDFLAGS=-L#{Formula["openssl@1.1"].lib}"
```

Comment:
This strikes me as more readable (and more consistent with other formulae):
```suggestion
    system "make", "install", "PREFIX=#{prefix}",
                              "CC=#{ENV.cc}",
                              "BUILD_TLS=yes",
                              "OPENSSL_CFLAGS=-I#{Formula["openssl@1.1"].include}",
                              "OPENSSL_LDFLAGS=-L#{Formula["openssl@1.1"].lib}"
```

Reply:
I don't care either way as long as it is readable. I usually try to avoid cosmetic changes in fixes.

Reply:
@SMillerDev Do you agree with the proposed style change?

Reply:
As long as the `tap syntax` check is happy I'm happy

Reply:
It's not a strong opinion. If you're happy with the way things read now, I don't mind keeping it either.